src/components/templates/AnimateMap/components/FirebarrelWidget/FirebarrelWidget.tsx

```diff
@@ -141,16 +134,11 @@ export const FirebarrelWidget: React.FC<FirebarrelWidgetProps> = ({
 
   const myVideo = useMemo(() => {
     return localParticipant && userWithId ? (
-      <div className="firebarrel-room__participant" key={localParticipant.sid}>
-        <VideoParticipant
-          participant={localParticipant}
-          participantUser={userWithId}
-          defaultMute={defaultMute}
-          isAudioEffectDisabled={isAudioEffectDisabled}
-        />
+      <div className="firebarrel-room__participant" key={localParticipant.id}>
```

Comment:
```suggestion
      <div className="FirebarrelWidget__participant" key={localParticipant.id}>
```

Still depending on `global.scss`? Fine:
```suggestion
      <div className="FirebarrelWidget__participant firebarrel-room__participant" key={localParticipant.id}>
```